docker/settings.py

```diff
@@ -207,9 +207,11 @@
     }
 }
 
-DEBUG = False
+DEBUG = True
```

Comment:
Is this a permanent change? Any exceptions will include most of the Django settings and other configuration in the output to the user.  

Also, if debug is False ALLOWED_HOSTS must be set or all requests will fail: https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-DEBUG

Reply:
No, not permanent .